store/index.ts

```diff
@@ -24,9 +38,10 @@ export interface RootState {
   };
 }
 export interface AppState extends RootState {
+  auth: AuthState;
```

Comment:
How is it used?

Reply:
It's not really related to this, but it's from the Nuxt-Auth plugin. The store exists, so it's just adding typings for it if needed.